solr/core/src/java/org/apache/solr/servlet/RateLimitManager.java

```diff
@@ -64,9 +66,12 @@ public class RateLimitManager implements ClusterPropertiesListener {
 
   private final String hostname;
 
-  public RateLimitManager(String hostname) {
+  private SolrMetricsContext solrMetricsContext;
+
+  public RateLimitManager(String hostname, SolrMetricsContext parentContext) {
     this.hostname = hostname;
     this.requestRateLimiterMap = new ConcurrentHashMap<>();
+    this.initializeMetrics(parentContext, "RateLimitManager");
```

Comment:
oops, forgot we're not even using `scope` in the callee method atm. No harm though.